<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceAdmin</class>
 <widget class="QMainWindow" name="AttendanceAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttendanceAdmin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>124141</width>
        <height>2123</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(6, 218, 149);
color: rgb(254, 254, 254);
border:none;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:20px;</string>
       </property>
       <property name="text">
        <string>考勤管理系统</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>4</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:30px;</string>
       </property>
       <property name="text">
        <string>LOGO</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">border:none;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(43, 49, 65);
color: #8a8a8a;
font:15px;
</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/image/lr.png);
image-position:left;
text-align:right;
</string>
            </property>
            <property name="text">
             <string>录入员工  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
image: url(:/image/qd.png);
image-position:left;
text-align:right;
</string>
            </property>
            <property name="text">
             <string>考勤明细  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/image/szt.png);
image-position:left;
text-align:right;
</string>
            </property>
            <property name="text">
             <string>列表统计  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/image/date.png);
image-position:left;
text-align:right;
</string>
            </property>
            <property name="text">
             <string>工作日历  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/image/cmd.png);
image-position:left;
text-align:right;
</string>
            </property>
            <property name="text">
             <string>控制台   </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>231</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="setFaceInfo" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="AttendanceTable" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
         <widget class="SignInStateWin" name="tab_3">
          <attribute name="title">
           <string>页</string>
          </attribute>
         </widget>
         <widget class="WorkDateWin" name="tab_4">
          <attribute name="title">
           <string>页</string>
          </attribute>
         </widget>
         <widget class="cmdWin" name="tab_5">
          <attribute name="title">
           <string>页</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>setFaceInfo</class>
   <extends>QWidget</extends>
   <header location="global">setfaceinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cmdWin</class>
   <extends>QWidget</extends>
   <header location="global">cmdwin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttendanceTable</class>
   <extends>QWidget</extends>
   <header location="global">attendancetable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorkDateWin</class>
   <extends>QWidget</extends>
   <header>workdatewin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SignInStateWin</class>
   <extends>QWidget</extends>
   <header>signinstatewin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
